maps_test.js

```diff
@@ -426,14 +410,14 @@ function makeTests(msgInfo, submessageCtor, suffix) {
 describe('mapsTest', () => {
   makeTests(
     {
-      constructor: proto.jspb.test.TestMapFields,
-      deserializeBinary: proto.jspb.test.TestMapFields.deserializeBinary
+      constructor: testbinary_pb.TestMapFields,
+      deserializeBinary: testbinary_pb.TestMapFields.deserializeBinary
     },
-    proto.jspb.test.MapValueMessage, '_Binary');
+    testbinary_pb.MapValueMessage, '_Binary');
   makeTests(
     {
-      constructor: proto.jspb.test.TestMapFieldsNoBinary,
+      constructor: testbinary_pb.TestMapFieldsNoBinary,
```

Comment:
`TestMapFieldsNoBinary` was imported separately and doesn't want a `testbinary_pb.` prefix.